tests/integ/test_cte.py

```diff
@@ -700,13 +700,14 @@ def test_table(session):
     check_result(
         session,
         df_result,
-        expect_cte_optimized=True,
+        expect_cte_optimized=False if session.sql_simplifier_enabled else True,
```

Comment:
Same comment about using a parameter to test for both settings of sql_simplifier_enabled.

Reply:
Same as above, all those tests are run with both sql simplify enabled and disabled in our daily test today.